Manito/Manito/Screens/Letter/Cell/LetterCollectionViewCell.swift

```diff
@@ -110,58 +115,73 @@ final class LetterCollectionViewCell: BaseCollectionViewCell {
     
     private func setupButtonAction() {
```

Comment:
p4;

```suggestion
    private func setupReportButtonAction() {
```

Cell에서는 버튼이 하나 밖에 없어서 현재 같이 사용해도 크게 이해하는데 어려움은 없지만, 변수에 `ReportButton`이라고 명시를 해두면 조금이나마 이해하는데 더 편하지 않을까 싶어 리뷰 남깁니다!

Reply:
이후에 Cell 내부에 다른 버튼이 생기고 해당 버튼에 액션을 넣게 될 수도 있다고 생각이 들어서 `setupButtonAction`으로 네이밍을 정했습니다. setupReportButtonAction으로 했을 시에 추후에 들어오는 버튼은 setup00ButtonAction으로 하던지, 다시 setupButtonAction으로 함수명을 변경해야할 거 같아서요.

최대한 `이후에 들어올 버튼을 유연하게 대비` + `현재 있는 버튼 액션을 정하는 함수`라는 의미를 모두 줄 수 있는 네이밍이라고 생각합니다. 그래도 네이밍을 변경하는 것이 더 좋을까요..? 👀 의견 주시면 감사드리겠습니다.

Reply:
아하 그렇군요! `setupButtonAction`에서 해당 뷰에 있는 버튼들을 다 설정하는거죠??
예시로 `CreateLetterView`의 `setupButtonAction`은 cancleButton과 sendButton을 둘 다 설정하는 것 처럼요!
그렇다면 현재 `setupButtonAction` 방식이 좋을 것 같습니다 ^_______^ 👍

Reply:
저도 setupButtonAction 이 좋다고 생각합니다!